application/forms/Account/ChangePasswordForm.php

```diff
@@ -34,6 +37,22 @@ public function init()
      */
     public function createElements(array $formData)
     {
+        $passwordPolicyObject = null;
+        $passwordPolicy = Config::app()->get(
+            'global',
+            'password_policy',
+            DefaultPasswordPolicy::class
+        );
+
+        if (isset($passwordPolicy)) {
```

Comment:
Now, after https://github.com/Icinga/icingaweb2/pull/5419#discussion_r2307135923, $passwordPolicy is always set.